week-1/E-strings/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+message = "Hi";
```

Comment:
Even though this compiles and runs ok, it's not the right way of initialising variables in JS - could you please fix that?